src/cargo/sources/git/known_hosts.rs

```diff
@@ -588,7 +589,14 @@ impl KnownHost {
         }
         for pattern in self.patterns.split(',') {
             let pattern = pattern.to_lowercase();
-            // FIXME: support * and ? wildcards
+            let is_glob = is_glob_pattern(&pattern);
+
+            if is_glob {
+                if let Ok(regex) = glob::Pattern::new(&pattern) {
+                    match_found |= regex.matches(&host);
```

Comment:
It is not regex anymore, so perhaps rename to `glob`?

Reply:
Oh! How could I miss this one! Thank you for pointing out :)